edxval/tests/test_api.py

```diff
@@ -334,6 +334,43 @@ def test_get_videos_for_ids_duplicates(self):
         videos = list(api.get_videos_for_ids([edx_video_id, edx_video_id]))
         self.assertEqual(len(videos), 1)
 
+    def test_get_videos_for_ids_sort(self):
+        fish_id = constants.VIDEO_DICT_FISH["edx_video_id"]
+        star_id = constants.VIDEO_DICT_STAR["edx_video_id"]
+        other_id = "other-video"
+        Video.objects.create(**constants.VIDEO_DICT_STAR)
+        Video.objects.create(
```

Comment:
To be consistent with the other tests in this file, consider using another already existing video in `constants` or moving the declaration of this video to the `constants` file.